package.json

```diff
@@ -47,43 +47,41 @@
     "e2e": "ng e2e"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.13.0",
-    "@angular-devkit/build-ng-packagr": "~0.13.0",
-    "@angular/cli": "~7.3.5",
-    "@angular/common": "^7.2.8",
-    "@angular/compiler": "^7.2.8",
-    "@angular/compiler-cli": "^7.2.8",
-    "@angular/core": "^7.2.8",
-    "@angular/forms": "^7.2.8",
-    "@angular/http": "^7.2.8",
-    "@angular/language-service": "^7.2.8",
-    "@angular/platform-browser": "^7.2.8",
-    "@angular/platform-browser-dynamic": "^7.2.8",
-    "@angular/router": "^7.2.8",
+    "@angular-devkit/build-angular": "~0.802.1",
+    "@angular-devkit/build-ng-packagr": "~0.802.1",
+    "@angular/cli": "~8.2.1",
+    "@angular/common": "^8.2.1",
+    "@angular/compiler": "^8.2.1",
+    "@angular/compiler-cli": "^8.2.1",
+    "@angular/core": "^8.2.1",
+    "@angular/forms": "^8.2.1",
+    "@angular/language-service": "^8.2.1",
+    "@angular/platform-browser": "^8.2.1",
+    "@angular/platform-browser-dynamic": "^8.2.1",
+    "@angular/router": "^8.2.1",
     "@earshinov/extract-scss-variables": "^0.1.1",
-    "@types/jasmine": "~2.8.8",
-    "@types/jasminewd2": "~2.0.3",
-    "@types/node": "~8.9.4",
-    "codelyzer": "~4.3.0",
-    "core-js": "^2.5.4",
-    "jasmine-core": "~2.99.1",
+    "@types/jasmine": "~3.4.0",
+    "@types/jasminewd2": "~2.0.6",
+    "@types/node": "~12.7.1",
+    "codelyzer": "^5.1.0",
+    "jasmine-core": "~3.4.0",
     "jasmine-spec-reporter": "~4.2.1",
-    "karma": "~3.0.0",
-    "karma-chrome-launcher": "~2.2.0",
-    "karma-coverage-istanbul-reporter": "~2.0.1",
-    "karma-jasmine": "~1.1.2",
-    "karma-jasmine-html-reporter": "^0.2.2",
-    "ng-packagr": "^4.1.0",
+    "karma": "~4.2.0",
+    "karma-chrome-launcher": "~3.0.0",
+    "karma-coverage-istanbul-reporter": "~2.1.0",
+    "karma-jasmine": "~2.0.1",
+    "karma-jasmine-html-reporter": "^1.4.2",
+    "ng-packagr": "^5.3.0",
     "node-sass": "^4.12.0",
     "protractor": "~5.4.0",
-    "rxjs": "~6.4.0",
-    "scss-bundle": "^2.5.0",
-    "ts-node": "~7.0.0",
-    "tsickle": ">=0.34.0",
-    "tslib": "^1.9.0",
-    "tslint": "~5.11.0",
-    "typescript": "~3.2.4",
-    "zone.js": "~0.8.29"
+    "rxjs": "~6.5.2",
+    "scss-bundle": "^2.5.1",
+    "ts-node": "~8.3.0",
+    "tsickle": "^0.36.0",
```

Comment:
Just a note. There is a warning

```
warning " > tsickle@0.36.0" has incorrect peer dependency "typescript@~3.4.1".
```

but `ng build` runs fine.

There is an open issue in `tsickle`: https://github.com/angular/tsickle/issues/1048